src/main/java/radiounit_executorandreceiver/EricssonLteCommandExecutor.java

```diff
@@ -0,0 +1,75 @@
+package radiounit_executorandreceiver;
```

Comment:
Minor: I 'd say let's just leave this in the `radiounit` package - might be confusing if we add too many packages

Reply:
Sounds good. I'll move them.